doc/sphinx/methods.rst

```diff
@@ -144,6 +144,66 @@ The LPA class available from cclib.method performs Löwdin population analysis a
     m = LPA(d)
     m.calculate()
 
+.. index::
+    single: methods; Bickelhaupt Population Analysis
+
+Bickelhaupt Population Analysis
+-------------------------------
+
+The Bickelhaupt class available from cclib.method performs Bickelhaupt population analysis that has been proposed in *Organometallics* 1996, 15, 13, 2923–2931. `doi:10.1021/om950966x <https://pubs.acs.org/doi/abs/10.1021/om950966x>`_
+
+The contribution of the a-th atomic orbital to the i-th molecular orbital in this method is written in terms of the molecular orbital coefficients, c, and the overlap matrix, S:
+
+.. math:: \Phi_{ai,\alpha} = \sum_b w_{ab,\alpha} c_{ai,\alpha} c_{bi,\alpha} S_{ab}
+
+where the weights :math:`w_{ab}` that are applied on the Mulliken atomic orbital contributions are defined as:
+
+.. math:: w_{ab,\alpha} = 2 \frac{\sum_k c_{ak,\alpha}^2}{\sum_i c_{ai,\alpha}^2 + \sum_j c_{bj,\alpha}^2}
```

Comment:
Related to https://github.com/cclib/cclib/pull/869#issuecomment-642323035, it is not immediately obvious to me how equation 11 in the paper is converted to this. Could you at least say that this leads to the same results?